### ISSUE TITLE
[Debian] The new release contains a python3-only line.

### ISSUE THREAD

**Issue by P-EB** [Koed00/django-q#291]
Hi,

In your new release, you introduced some python3-only code:

```
  File "django_q/queues.py", line 51, in __init__
    super(Queue, self).__init__(*args, ctx=multiprocessing.get_context(), **kwargs)
```

[Here](https://github.com/Koed00/django-q/files/1698329/0003-Use-conditional-on-a-python3-only-part-of-the-code.txt) is a patch I made.

Cheers!

**Comment by Eagllus** [comment]
@P-EB thank you for the patch.

**Comment by P-EB** [comment]
@Eagllus you're welcome.

Should I open a PR or let you handle it from there?

**Comment by Eagllus** [comment]
@P-EB I included your fix in pr #292 🙂

**Comment by P-EB** [comment]
Perfect!
